content/en/docs/contribute/participate/pr-wranglers.md

```diff
@@ -26,6 +26,7 @@ Each day in a week-long shift as PR Wrangler:
     - If you need to verify content, comment on the PR and request more details.
     - Assign relevant `sig/` label(s).
     - If needed, assign reviewers from the `reviewers:` block in the file's front matter.
+    - You can also tag a SIG for a review by commenting `@kubernetes/<sig>-pr-reviews` on the PR.
```

Comment:
nit: This doesn't always work. GitHub autocomplete is a way to check though (you need to be an org member to use that autocompletion, I think).

All PR wranglers are org members.